Makefile

```
@@ -1,36 +1,46 @@
 # Builds PEP files to HTML using sphinx
 
 PYTHON=python3
-VENV_DIR=venv
+VENVDIR=venv
```

I would recommend making it `.venv` to hide it from tooling.

```suggestion
VENVDIR=.venv
```